relay-kafka/src/config.rs

```diff
@@ -85,41 +85,52 @@ impl KafkaTopic {
 
 /// Configuration for topics.
 #[derive(Serialize, Deserialize, Debug)]
-#[serde(default)]
+#[serde(default, deny_unknown_fields)]
 pub struct TopicAssignments {
     /// Simple events topic name.
+    #[serde(alias = "ingest-events")]
```

Comment:
alias is supposed to be temporary, while we cut over config in ops. Later this should change to rename